<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> publications | Lingchao Mao </title> <meta name="author" content="Lingchao Mao"> <meta name="description" content="A simple, whitespace theme for academics. Based on [*folio](https://github.com/bogoli/-folio) design. "> <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" href="/assets/css/scholar-icons.css?62b2ac103a88034e6882a5be5f3e2772"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="/assets/img/prof_pic.jpg?6b31cb03b8c81758747d5504d9751e70"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://lingchm.github.io/publications/"> <script src="/assets/js/theme.js?9a0c749ec5240d9cda97bc72359a72c0"></script> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script>initTheme();</script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"> <span class="font-weight-bold">Lingchao</span> Mao </a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about </a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">publications <span class="sr-only">(current)</span> </a> </li> <li class="nav-item "> <a class="nav-link" href="/research/">research </a> </li> <li class="nav-item "> <a class="nav-link" href="/tools/">tools </a> </li> <li class="nav-item "> <a class="nav-link" href="/personal/">personal </a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="ti ti-sun-moon" id="light-toggle-system"></i> <i class="ti ti-moon-filled" id="light-toggle-dark"></i> <i class="ti ti-sun-filled" id="light-toggle-light"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title">publications</h1> <p class="post-description"></p> </header> <article> <div class="publications"> <h2 class="bibliography">2025</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100">TMI</abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/bivesnet-480.webp 480w,/assets/img/publication_preview/bivesnet-800.webp 800w,/assets/img/publication_preview/bivesnet-1400.webp 1400w," type="image/webp" sizes="200px"></source> <img src="/assets/img/publication_preview/bivesnet.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="bivesnet.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="mao2024vessel" class="col-sm-8"> <div class="title">BiVesNet: A Novel Hepatic/Portal Vein Segmentation Model for Diagnosis and Preoperative Liver Surgical Planning</div> <div class="author"> Mian Wu, Tuo Ren, <em>Lingchao Mao</em>, Yonghui Huang, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Jing Li' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>IEEE Transactions on Medical Imaging (under review)</em>, 2025 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="abstract hidden"> <p>Accurate segmentation of Hepatic Vessels (HV) and Portal Vessels (PV) from CT images is crucial for the diagnosis and surgical planning of liver diseases. However, several challenges hinder automated segmentation solutions, including the complex vascular structure, the small scale and irregularity of vessels, low contrast and noisy background, and the closely interleaved HV/PV structures. In this paper, we introduce BiVesNet, the first learning-based model for automated and reliable HV/PV vessel segmentation. Our model consists of two stages: In Stage I, we perform single-class vessel segmentation using a hierarchical CNN-Transformer architecture that incorporates Gaussian-prior relative positional embeddings and a specialized attention mechanism to effectively capture local dependencies in vessel structures and differentiate the background with vessel. In Stage II, we introduce a novel topology-based algorithm for HV/PV vessel separation, which effectively handles residual vessels and discontinuities from the previous stage by leveraging Graph Attention Networks (GAT) to recover the topological relationships between the residuals and the main vessel trees. By pre-training and fine-tuning our models on three,publicly available CT datasets and one high-quality clinical dataset, BiVesNet consistently outperform state-of-the-art methods for liver vessel segmentation. Remarkably, our two-class HV/PV segmentation achieves performance on par with current semi-automatic clinical protocols. These results represent a significant step toward advancing fully automated HV/PV vessel segmentation tools, which can provide substantial clinical value for liver diagnostics and surgical planning.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">mao2024vessel</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{BiVesNet: A Novel Hepatic/Portal Vein Segmentation Model for Diagnosis and Preoperative Liver Surgical Planning}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Wu, Mian and Ren, Tuo and Mao, Lingchao and Huang, Yonghui and Li, Jing}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{IEEE Transactions on Medical Imaging (under review)}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2025}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100">IJDS</abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/mmfl-480.webp 480w,/assets/img/publication_preview/mmfl-800.webp 800w,/assets/img/publication_preview/mmfl-1400.webp 1400w," type="image/webp" sizes="200px"></source> <img src="/assets/img/publication_preview/mmfl.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="mmfl.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="mao2024mmfl" class="col-sm-8"> <div class="title">Supervised Multi-Modal Fission Learning</div> <div class="author"> <em>Lingchao Mao</em>, Qi Wang, Yi Su, Fleming Lure, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Jing Li' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>arXiv preprint arXiv:2409.20559 (under review)</em>, 2025 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="abstract hidden"> <p>Learning from multi-modal datasets can leverage complementary information and lead to improved performance for prediction tasks. To account for feature correlations in high-dimensional datasets, a commonly used strategy is the latent variable approach. Several latent variable methods in the literature have been proposed for multi-modal datasets, however, these methods either focus on extracting the shared component across all modalities or extracting a shared component and individual components specific to each modality. To address this gap, we propose a Multi-Modal Fission Learning (MMFL) model that simultaneously identifies globally joint, partially joint, and individual components underlying the features of multi-modal datasets. Unlike existing latent variable methods, MMFL uses supervision from labels to identify predictive latent components and has a natural extension to incorporate incomplete multi-modal data. In our simulation studies, MMFL outperformed a variety of existing multi-modal algorithms under both complete modality and incomplete modality settings. We applied MMFL to a real-world case study for early prediction of Alzheimer’s Disease using multi-modal neuroimaging (MRI and PET) and genetic data (SNP) from the Alzheimer’s Disease Neuroimaging Initiative (ADNI) dataset. MMFL provided more accurate predictions and better insights for understanding within and across modality correlations compared to existing methods.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">mao2024mmfl</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Supervised Multi-Modal Fission Learning}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Mao, Lingchao and Wang, Qi and Su, Yi and Lure, Fleming and Li, Jing}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{arXiv preprint arXiv:2409.20559 (under review)}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2025}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100">IISE Trans. Health</abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/liver2024-480.webp 480w,/assets/img/publication_preview/liver2024-800.webp 800w,/assets/img/publication_preview/liver2024-1400.webp 1400w," type="image/webp" sizes="200px"></source> <img src="/assets/img/publication_preview/liver2024.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="liver2024.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="mao2024liver" class="col-sm-8"> <div class="title">A Weakly Supervised Approach with Knowledge-Informed Label Smoothing for Liver Tumor Segmentation</div> <div class="author"> Hairong Wang, <em>Lingchao Mao</em>, Zihan Zhang, and Jing Li </div> <div class="periodical"> <em>arXiv preprint arXiv:2410.10005 (under review)</em>, 2025 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="abstract hidden"> <p>Liver cancer is a leading cause of mortality worldwide, and accurate CT-based tumor segmentation is essential for diagnosis and treatment. Manual delineation is time-intensive, prone to variability, and highlights the need for reliable automation. While deep learning has shown promise for automated liver segmentation, precise liver tumor segmentation remains challenging due to the heterogeneous nature of tumors, imprecise tumor margins, and limited labeled data. We present a novel holistic weakly supervised framework that integrates clinical knowledge to address these challenges with (1) A knowledge-informed label smoothing technique that leverages clinical data to generate smooth labels, which regularizes model training reducing the risk of overfitting and enhancing model performance; (2) A global and local-view segmentation framework, breaking down the task into two simpler sub-tasks, allowing optimized preprocessing and training for each; and (3) Pre- and post-processing pipelines customized to the challenges of each subtask, which enhances tumor visibility and refines tumor boundaries. We evaluated the proposed method on the HCC-TACE-Seg dataset and showed that these three key components complementarily contribute to the improved performance. Lastly, we prototyped a tool for automated liver tumor segmentation and diagnosis summary generation called MedAssistLiver. The app and code are published at https://github.com/lingchm/medassistliver-cancer.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">mao2024liver</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{A Weakly Supervised Approach with Knowledge-Informed Label Smoothing for Liver Tumor Segmentation}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Wang, Hairong and Mao, Lingchao and Zhang, Zihan and Li, Jing}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{arXiv preprint arXiv:2410.10005 (under review)}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2025}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100">IEEE-TASE</abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/mkd-480.webp 480w,/assets/img/publication_preview/mkd-800.webp 800w,/assets/img/publication_preview/mkd-1400.webp 1400w," type="image/webp" sizes="200px"></source> <img src="/assets/img/publication_preview/mkd.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="mkd.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="mao2024mkd" class="col-sm-8"> <div class="title">A Cross-Modal Mutual Knowledge Distillation Framework for Alzheimer’s Disease Diagnosis: Addressing Incomplete Modalities</div> <div class="author"> Mingu Kwak, <em>Lingchao Mao</em>, Zhiyang Zheng, Yi Su, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Lure Fleming, Jing Li' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>medRxiv (under review)</em>, 2025 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1101/2023.08.24.23294574" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="abstract hidden"> <p>Early detection of Alzheimer’s Disease (AD) is crucial for timely interventions and optimizing treatment outcomes. Despite the promise of integrating multimodal neuroimages such as MRI and PET, handling datasets with incomplete modalities remains under-researched. This phenomenon, however, is common in real-world scenarios as not every patient has all modalities due to practical constraints such as cost, access, and safety concerns. We propose a deep learning framework employing cross-modal Mutual Knowledge Distillation (MKD) to model different sub-cohorts of patients based on their available modalities. In MKD, the multimodal model (e.g., MRI and PET) serves as a teacher, while the single-modality model (e.g., MRI only) is the student. Our MKD framework features three components: a Modality-Disentangling Teacher (MDT) model designed through information disentanglement, a student model that learns from classification errors and MDT’s knowledge, and the teacher model enhanced via distilling the student’s single-modal feature extraction capabilities. Moreover, we show the effectiveness of the proposed method through theoretical analysis and validate its performance with simulation studies. In addition, our method is demonstrated through a case study with Alzheimer’s Disease Neuroimaging Initiative (ADNI) datasets, underscoring the potential of artificial intelligence in addressing incomplete multimodal neuroimaging datasets and advancing early AD detection.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">mao2024mkd</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{A Cross-Modal Mutual Knowledge Distillation Framework for Alzheimer's Disease Diagnosis: Addressing Incomplete Modalities}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Kwak, Mingu and Mao, Lingchao and Zheng, Zhiyang and Su, Yi and Fleming, Lure and Li, Jing}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{medRxiv (under review)}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2025}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1101/2023.08.24.23294574}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100">Headache</abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/headache2024-480.webp 480w,/assets/img/publication_preview/headache2024-800.webp 800w,/assets/img/publication_preview/headache2024-1400.webp 1400w," type="image/webp" sizes="200px"></source> <img src="/assets/img/publication_preview/headache2024.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="headache2024.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="mao2024headache" class="col-sm-8"> <div class="title">Identifying and Predicting Headache Trajectories Amongst Those with Acute Post-Traumatic Headache</div> <div class="author"> <em>Lingchao Mao</em>, Jing Li, Todd Schwedt, Teresa Wu, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Katherine Ross, Gina Dumkrieger, Dani Smith, Catherine D Chong' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em>Headache (under review)</em>, 2025 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="abstract hidden"> <p>Background: Post-traumatic headache (PTH) is a common symptom following mild traumatic brain injury (mTBI). Currently, there is no identified way to accurately predict if, when, and at what pace a person will have PTH improvement. In our prior studies, we focused on predicting headache improvement at three months post-mTBI. However, that approach may overlook individual differences in how headaches evolve over time. Objective: This study aims to identify individual subgroups based on their headache trajectories and to develop machine learning (ML) models for early prediction of headache evolution. Methods: Participants with acute PTH completed a daily electronic headache diary (eDIARY) over three months, recording their headache-related symptoms. Tensor decomposition was utilized to extract latent factors underlying the time-varying symptoms. We applied clustering techniques on the latent factors to identify patient subgroups with varying headache improvement trajectory. Next, we developed a ML method to classify each individual into a headache trajectory subgroup as early as possible within the three-month interval. Results: 73 individuals with acute PTH (mean age=44.8, SD=14.0; 50 females/23 males) were enrolled between 0-59 days post-mTBI. Data from 54 individuals were used as the developmental cohort for model training and 19 individuals were used as the test cohort for model evaluation. Tensor decomposition extracted two latent factors: one factor representing the ‘overall state’ of PTH severity and disability and the other representing the ‘improvement state’ of these symptoms over three months. Clustering identified four patient subgroups with distinct headache evolution trajectories: 1) severe symptoms without improvement, 2) severe symptoms with mild improvement, 3) milder symptoms with substantial improvement, 4) mildest symptoms with minimal improvement .The proposed ML model achieved 0.80 cross-validation accuracy in classifying individuals with PTH into subgroups for the training cohort and 0.84 accuracy for the test cohort. Notably, the model required only the first two weeks of headache data to accurately identify the subgroup with the mildest headaches, three additional weeks to identify the subgroup with the most severe headaches and no improvement in three months, and two additional weeks to distinguish the remaining subgroups. Conclusions: This study identified subgroups of individuals with acute PTH with distinct headache improvement trajectories. The proposed ML method accurately classified individuals into these subgroups using the minimally necessary early headache data for each person, including detecting the subgroup with the mildest headaches at two weeks. This approach could offer an estimated forecast of headache burden over time and could assist clinicians with determining treatment needs and eligibility for PTH clinical trials. </p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">mao2024headache</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Identifying and Predicting Headache Trajectories Amongst Those with Acute Post-Traumatic Headache}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Mao, Lingchao and Li, Jing and Schwedt, Todd and Wu, Teresa and Ross, Katherine and Dumkrieger, Gina and Smith, Dani and Chong, Catherine D}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Headache (under review)}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2025}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100">medRxiv</abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/revealing-480.webp 480w,/assets/img/publication_preview/revealing-800.webp 800w,/assets/img/publication_preview/revealing-1400.webp 1400w," type="image/webp" sizes="200px"></source> <img src="/assets/img/publication_preview/revealing.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="revealing.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="lewis2023revealing" class="col-sm-8"> <div class="title">Revealing the biology behind MRI signatures in high grade glioma</div> <div class="author"> Erika M Lewis, <em>Lingchao Mao</em>, Lujia Wang, Kristin R Swanson, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'Ramon F Barajas, Jing Li, Nhan L Tran, Leland S Hu, Christopher L Plaisier' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> <em>medRxiv (under review)</em>, 2025 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="abstract hidden"> <p>Magnetic resonance imaging (MRI) measurements are routinely collected during the treatment of high-grade gliomas (HGGs) to characterize tumor boundaries and guide surgical tumor resection. Using spatially matched MRI and transcriptomics we discovered HGG tumor biology captured by MRI measurements. We strategically overlaid the spatially matched omics characterizations onto a pre-existing transcriptional map of glioblastoma multiforme (GBM) to enhance the robustness of our analyses. We discovered that T1+C measurements, designed to capture vasculature and blood brain barrier (BBB) breakdown and subsequent contrast extravasation, also indirectly reveal immune cell infiltration. The disruption of the vasculature and BBB within the tumor creates a permissive infiltrative environment that enables the transmigration of anti-inflammatory macrophages into tumors. These relationships were validated through histology and enrichment of genes associated with immune cell transmigration and proliferation. Additionally, T2-weighted (T2W) and mean diffusivity (MD) measurements were associated with angiogenesis and validated using histology and enrichment of genes involved in neovascularization. Furthermore, we establish an unbiased approach for identifying additional linkages between MRI measurements and tumor biology in future studies, particularly with the integration of novel MRI techniques. Lastly, we illustrated how noninvasive MRI can be used to map HGG biology spatially across a tumor, and this provides a platform to develop diagnostics, prognostics, or treatment efficacy biomarkers to improve patient outcomes.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">lewis2023revealing</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Revealing the biology behind MRI signatures in high grade glioma}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Lewis, Erika M and Mao, Lingchao and Wang, Lujia and Swanson, Kristin R and Barajas, Ramon F and Li, Jing and Tran, Nhan L and Hu, Leland S and Plaisier, Christopher L}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{medRxiv (under review)}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{2023--12}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2025}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Cold Spring Harbor Laboratory Press}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> </ol> <h2 class="bibliography">2024</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100">IEEE-TASE</abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/kiml-480.webp 480w,/assets/img/publication_preview/kiml-800.webp 800w,/assets/img/publication_preview/kiml-1400.webp 1400w," type="image/webp" sizes="200px"></source> <img src="/assets/img/publication_preview/kiml.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="kiml.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="mao2024knowledge" class="col-sm-8"> <div class="title">Knowledge-Informed Machine Learning for Cancer Diagnosis and Prognosis: A review</div> <div class="author"> <em>Lingchao Mao</em>, Hairong Wang, Leland S Hu, Nhan L Tran, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Peter D Canoll, Kristin R Swanson, Jing Li' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>IEEE Transactions on Automation Science and Engineering</em>, 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1109/TASE.2024.3515839" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="abstract hidden"> <p>Cancer remains one of the most challenging diseases to treat in the medical field. Machine learning has enabled in-depth analysis of rich multi-omics profiles and medical imaging for cancer diagnosis and prognosis. Despite these advancements, machine learning models face challenges stemming from limited labeled sample sizes, the intricate interplay of high-dimensionality data types, the inherent heterogeneity observed among patients and within tumors, and concerns about interpretability and consistency with existing biomedical knowledge. One approach to surmount these challenges is to integrate biomedical knowledge into data-driven models, which has proven potential to improve the accuracy, robustness, and interpretability of model results. Here, we review the state-of-the-art machine learning studies that adopted the fusion of biomedical knowledge and data, termed knowledge-informed machine learning, for cancer diagnosis and prognosis. Emphasizing the properties inherent in four primary data types including clinical, imaging, molecular, and treatment data, we highlight modeling considerations relevant to these contexts. We provide an overview of diverse forms of knowledge representation and current strategies of knowledge integration into machine learning pipelines with concrete examples. We conclude the review article by discussing future directions to advance cancer research through knowledge-informed machine learning.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">mao2024knowledge</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Knowledge-Informed Machine Learning for Cancer Diagnosis and Prognosis: A review}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Mao, Lingchao and Wang, Hairong and Hu, Leland S and Tran, Nhan L and Canoll, Peter D and Swanson, Kristin R and Li, Jing}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{IEEE Transactions on Automation Science and Engineering}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{0}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1109/TASE.2024.3515839}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100">JMIR</abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/jmir2024-480.webp 480w,/assets/img/publication_preview/jmir2024-800.webp 800w,/assets/img/publication_preview/jmir2024-1400.webp 1400w," type="image/webp" sizes="200px"></source> <img src="/assets/img/publication_preview/jmir2024.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="jmir2024.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="mao2024sodium" class="col-sm-8"> <div class="title">Tracking Influencers in Policy Field on Social Media: A Global Longitudinal Study of Dietary Sodium Reduction Posts, 2006-2022</div> <div class="author"> Alana Montoya, <em>Lingchao Mao</em>, Adam Drewnowski, Joshua Chen, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Ella Shi, Aileen Liang, Bryan Weiner, Yanfang Su' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em>Journal of Medical Internet Research</em>, 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.2196/54506" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="abstract hidden"> <p>Background: Excessive sodium intake is a major concern for global public health. Despite multiple dietary guidelines, population sodium intakes are above recommended levels. Lack of health literacy could be one contributing issue and contemporary health literacy is largely shaped by social media. Objective: We aim to quantify the posting behaviors and influence patterns on dietary sodium-related content by influencers in policy field on X (formerly Twitter) across time. Methods: We first identified X users with a scope of work related to dietary sodium and retrieved their posts (formerly Tweets) from 2006 to 2022. Users were categorized into the policy groups of outer-setting organization, inner-setting organization, or individual, based on their role in the conceptual policy field. Network analysis was used to analyze interactions among users and identify the top influencers in each policy group. A four-dimensional influence framework was applied to measure the overall influence, activity, priority, originality, and popularity scores. These measures were used to reveal the user-level, group-level, and temporal patterns of sodium-related influence. Results: We identified 78 users with content related to dietary sodium, with 1,099,605 posts in total and 14,732 dietary sodium posts. There was an increasing volume of sodium posts from 2010 to 2015; however, the trend has been decreasing since 2016, especially among outer-setting organizations. The top influencers from the three policy groups were the World Health Organization (WHO), American Heart Association (AHA), and Tom Frieden, with a total public engagement of 55,593, 26,395, and 12,672, respectively. The WHO and Simon Capewell ranked the highest in activity; the World Action on Salt, Sugar and Health (WASSH) and Action on Salt had the highest priority for dietary sodium content; General Mills, Tom Frieden, and Dariush Mozaffarian had the highest originality; WHO, Tom Frieden, and Harvard University School of Medicine received the highest popularity. The top influencers frequently interacted among themselves, especially those with similar initiatives and partnerships. Outer-setting organizations tend to interact with more users in the network compared to inner-setting organizations and individuals, while inner-setting organizations tend to receive more interactions from other users in the network than the other two groups. Monthly patterns showed a significant peak in the number of sodium posts in March compared with other months. Conclusion: Despite the increased usage of social media, recent trends of sodium intake education on social media are decreasing and the priority of sodium among other topics is low. To improve policy implementation effectiveness and meet recommended dietary targets, there is an increasing need for health leaders to consistently and collectively advocate for sodium intake reduction on social media.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">mao2024sodium</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Tracking Influencers in Policy Field on Social Media: A Global Longitudinal Study of Dietary Sodium Reduction Posts, 2006-2022}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Montoya, Alana and Mao, Lingchao and Drewnowski, Adam and Chen, Joshua and Shi, Ella and Liang, Aileen and Weiner, Bryan and Su, Yanfang}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Journal of Medical Internet Research}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.2196/54506}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> </ol> <h2 class="bibliography">2023</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100">IEEE-TASE</abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/wstl-480.webp 480w,/assets/img/publication_preview/wstl-800.webp 800w,/assets/img/publication_preview/wstl-1400.webp 1400w," type="image/webp" sizes="200px"></source> <img src="/assets/img/publication_preview/wstl.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="wstl.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="mao2023wstl" class="col-sm-8"> <div class="title">Weakly Supervised Transfer Learning with Application in Precision Medicine</div> <div class="author"> <em>Lingchao Mao</em>, Lujia Wang, Leland Hu, Jennifer M Eschbacher, and <span class="more-authors" title="click to view 9 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '9 more authors' ? 'Gustavo D. Leon, Kyle W. Singleton, Lee A. Curtin, Javier Urcuyo, Chris Sereduk, Nhan L. Tran, Andrea Hawkins-Daarud, Kristin R. Swanson, Jing Li' : '9 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">9 more authors</span> </div> <div class="periodical"> <em>IEEE Transactions on Automation Science and Engineering</em>, 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="award btn btn-sm z-depth-0" role="button">Award</a> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1109/TASE.2023.3323773" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="award hidden d-print-inline"> <p></p> <p>IISE DAIS Best Student Paper, 2022; QPRC Best Poster Runner-up, 2021</p> </div> <div class="abstract hidden"> <p>Precision medicine aims to provide diagnosis and treatment accounting for individual differences. To develop machine learning models in support of precision medicine, personalized models are expected to have better performance than one-model-fits-all approaches. A significant challenge, however, is the limited number of labeled samples that can be collected from each individual due to practical constraints. Transfer Learning (TL) addresses this challenge by leveraging the information of other patients with the same disease (i.e., the source domain) when building a personalized model for each patient (i.e., the target domain). We propose Weakly-Supervised Transfer Learning (WS-TL) to tackle two challenges that existing TL algorithms do not address well: (i) the target domain has only a few or even no labeled samples; (ii) how to integrate domain knowledge into themTL design. We design a novel mathematical framework of WS-TL to learn a model for the target domain based on paired samples whose order relationships are inferred from domain knowledge, while at the same time integrating labeled samples in the source domain for transfer learning. Also, we propose an efficient active sampling strategy to select informative paired samples. Theoretical properties were investigated. Finally, we present a real-world application in precision medicine of brain cancer, where WS-TL is used to build personalized patient models to predict Tumor Cell Density (TCD) distribution across the brain based on MRI images. WS-TL has the highest accuracy compared to a variety of existing TL algorithms. The predicted TCD map for each patient can help facilitate individually optimized treatment.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">mao2023wstl</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Weakly Supervised Transfer Learning with Application in Precision Medicine}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Mao, Lingchao and Wang, Lujia and Hu, Leland and Eschbacher, Jennifer M and Leon, Gustavo D. and Singleton, Kyle W. and Curtin, Lee A. and Urcuyo, Javier and Sereduk, Chris and Tran, Nhan L. and Hawkins-Daarud, Andrea and Swanson, Kristin R. and Li, Jing}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{IEEE Transactions on Automation Science and Engineering}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1109/TASE.2023.3323773}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100">Cephalalgia</abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/cephalalgia_2023-480.webp 480w,/assets/img/publication_preview/cephalalgia_2023-800.webp 800w,/assets/img/publication_preview/cephalalgia_2023-1400.webp 1400w," type="image/webp" sizes="200px"></source> <img src="/assets/img/publication_preview/cephalalgia_2023.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="cephalalgia_2023.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="mao2023questionnaire" class="col-sm-8"> <div class="title">Questionnaire and structural imaging data accurately predict headache improvement in patients with acute post-traumatic headache attributed to mild traumatic brain injury</div> <div class="author"> <em>Lingchao Mao</em>, Jing Li, Todd J Schwedt, Visar Berisha, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'Devin Nikjou, Teresa Wu, Gina M Dumkrieger, Katherine B Ross, Catherine D Chong' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> <em>Cephalalgia</em>, 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1177/03331024231172736" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="/assets/pdf/headache_2023.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>Our prior work demonstrated that questionnaires assessing psychosocial symptoms have utility for predicting improvement in patients with acute post-traumatic headache following mild traumatic brain injury. In this cohort study, we aimed to determine whether prediction accuracy can be refined by adding structural magnetic resonance imaging (MRI) brain measures to the model. Adults with acute post-traumatic headache (enrolled 0–59 days post-mild traumatic brain injury) underwent T1-weighted brain MRI and completed three questionnaires (Sports Concussion Assessment Tool, Pain Catastrophizing Scale, and the Trait Anxiety Inventory Scale). Individuals with post-traumatic headache completed an electronic headache diary allowing for determination of headache improvement at three- and at six-month follow-up. Questionnaire and MRI measures were used to train prediction models of headache improvement and headache trajectory. Forty-three patients with post-traumatic headache (mean age = 43.0, SD = 12.4; 27 females/16 males) and 61 healthy controls were enrolled (mean age = 39.1, SD = 12.8; 39 females/22 males). The best model achieved cross-validation Area Under the Curve of 0.801 and 0.805 for predicting headache improvement at three and at six months. The top contributing MRI features for the prediction included curvature and thickness of superior, middle, and inferior temporal, fusiform, inferior parietal, and lateral occipital regions. Patients with post-traumatic headache who did not improve by three months had less thickness and higher curvature measures and notably greater baseline differences in brain structure vs. healthy controls (thickness: p &lt; 0.001, curvature: p = 0.012) than those who had headache improvement. A model including clinical questionnaire data and measures of brain structure accurately predicted headache improvement in patients with post-traumatic headache and achieved improvement compared to a model developed using questionnaire data alone.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">mao2023questionnaire</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Questionnaire and structural imaging data accurately predict headache improvement in patients with acute post-traumatic headache attributed to mild traumatic brain injury}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Mao, Lingchao and Li, Jing and Schwedt, Todd J and Berisha, Visar and Nikjou, Devin and Wu, Teresa and Dumkrieger, Gina M and Ross, Katherine B and Chong, Catherine D}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Cephalalgia}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{43}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{5}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{SAGE Publications Sage UK: London, England}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1177/03331024231172736}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://journals.sagepub.com/doi/full/10.1177/03331024231172736}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100">Headache</abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/headache-480.webp 480w,/assets/img/publication_preview/headache-800.webp 800w,/assets/img/publication_preview/headache-1400.webp 1400w," type="image/webp" sizes="200px"></source> <img src="/assets/img/publication_preview/headache.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="headache.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="mao2023developing" class="col-sm-8"> <div class="title">Developing multivariable models for predicting headache improvement in patients with acute post-traumatic headache attributed to mild traumatic brain injury: A preliminary study</div> <div class="author"> <em>Lingchao Mao</em>, Gina Dumkrieger, Dohyun Ku, Katherine Ross, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Visar Berisha, Todd J Schwedt, Jing Li, Catherine D Chong' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em>Headache: The Journal of Head and Face Pain</em>, 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1111/head.14450" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="/assets/pdf/cephalalgia_2023.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>Post-traumatic headache (PTH) is a common symptom after mild traumatic brain injury (mTBI). Although there have been several studies that have used clinical features of PTH to attempt to predict headache recovery, currently no accurate methods exist for predicting individuals’ improvement from acute PTH. This study investigated the utility of clinical questionnaires for predicting (i) headache improvement at 3 and 6 months, and (ii) headache trajectories over the first 3 months. We conducted a clinic-based observational longitudinal study of patients with acute PTH who completed a battery of clinical questionnaires within 0–59 days post-mTBI. The battery included headache history, symptom evaluation, cognitive tests, psychological tests, and scales assessing photosensitivity, hyperacusis, insomnia, cutaneous allodynia, and substance use. Each participant completed a web-based headache diary, which was used to determine headache improvement. Thirty-seven participants with acute PTH (mean age = 42.7, standard deviation [SD] = 12.0; 25 females/12 males) completed questionnaires at an average of 21.7 (SD = 13.1) days post-mTBI. The classification of headache improvement or non-improvement at 3 and 6 months achieved cross-validation area under the curve (AUC) of 0.72 and 0.84. Sub-models trained using only the top five features still achieved 0.72 and 0.77 AUC. The top five contributing features were from three questionnaires: Pain Catastrophizing Scale total score and helplessness sub-domain score; Sports Concussion Assessment Tool Symptom Evaluation total score and number of symptoms; and the State-Trait Anxiety Inventory score. The functional regression model achieved for modeling headache trajectory over the first 3 months. Questionnaires completed following mTBI have good utility for predicting headache improvement at 3 and 6 months in the future as well as the evolving headache trajectory. Reducing the battery to only three questionnaires, which assess post-concussive symptom load and biopsychosocialecologic factors, was helpful to determine a reasonable prediction accuracy for headache improvement.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">mao2023developing</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Developing multivariable models for predicting headache improvement in patients with acute post-traumatic headache attributed to mild traumatic brain injury: A preliminary study}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Mao, Lingchao and Dumkrieger, Gina and Ku, Dohyun and Ross, Katherine and Berisha, Visar and Schwedt, Todd J and Li, Jing and Chong, Catherine D}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Headache: The Journal of Head and Face Pain}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{63}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{136--145}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Wiley Online Library}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1111/head.14450}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://pubmed.ncbi.nlm.nih.gov/36651586/}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100">JMIR</abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/twitter_sodium-480.webp 480w,/assets/img/publication_preview/twitter_sodium-800.webp 800w,/assets/img/publication_preview/twitter_sodium-1400.webp 1400w," type="image/webp" sizes="200px"></source> <img src="/assets/img/publication_preview/twitter_sodium.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="twitter_sodium.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="mao20234d" class="col-sm-8"> <div class="title">A 4D Theoretical Framework for Measuring Topic-Specific Influence on Twitter: Development and Usability Study on Dietary Sodium Tweets</div> <div class="author"> <em>Lingchao Mao</em>, Emily Chu, Jinghong Gu, Tao Hu, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Bryan J Weiner, Yanfang Su' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>Journal of Medical Internet Research</em>, 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.2196/45897" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="abstract hidden"> <p>Background: Social media has emerged as a prominent approach for health education and promotion. However, it is challenging to understand how to best promote health-related information on social media platforms such as Twitter. Despite commercial tools and prior studies attempting to analyze influence, there is a gap to fill in developing a publicly accessible and consolidated framework to measure influence and analyze dissemination strategies. Objective: We aimed to develop a theoretical framework to measure topic-specific user influence on Twitter and to examine its usability by analyzing dietary sodium tweets to support public health agencies in improving their dissemination strategies. Methods: We designed a consolidated framework for measuring influence that can capture topic-specific tweeting behaviors. The core of the framework is a summary indicator of influence decomposable into 4 dimensions: activity, priority, originality, and popularity. These measures can be easily visualized and efficiently computed for any Twitter account without the need for private access. We demonstrated the proposed methods by using a case study on dietary sodium tweets with sampled stakeholders and then compared the framework with a traditional measure of influence. Results: More than half a million dietary sodium tweets from 2006 to 2022 were retrieved for 16 US domestic and international stakeholders in 4 categories, that is, public agencies, academic institutions, professional associations, and experts. We discovered that World Health Organization, American Heart Association, Food and Agriculture Organization of the United Nations (UN-FAO), and World Action on Salt (WASH) were the top 4 sodium influencers in the sample. Each had different strengths and weaknesses in their dissemination strategies, and 2 stakeholders with similar overall influence, that is, UN-FAO and WASH, could have significantly different tweeting patterns. In addition, we identified exemplars in each dimension of influence. Regarding tweeting activity, a dedicated expert published more sodium tweets than any organization in the sample in the past 16 years. In terms of priority, WASH had more than half of its tweets dedicated to sodium. UN-FAO had both the highest proportion of original sodium tweets and posted the most popular sodium tweets among all sampled stakeholders. Regardless of excellence in 1 dimension, the 4 most influential stakeholders excelled in at least 2 out of 4 dimensions of influence. Conclusions: Our findings demonstrate that our method not only aligned with a traditional measure of influence but also advanced influence analysis by analyzing the 4 dimensions that contribute to topic-specific influence. This consolidated framework provides quantifiable measures for public health entities to understand their bottleneck of influence and refine their social media campaign strategies. Our framework can be applied to improve the dissemination of other health topics as well as assist policy makers and public campaign experts to maximize population impact.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">mao20234d</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{A 4D Theoretical Framework for Measuring Topic-Specific Influence on Twitter: Development and Usability Study on Dietary Sodium Tweets}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Mao, Lingchao and Chu, Emily and Gu, Jinghong and Hu, Tao and Weiner, Bryan J and Su, Yanfang}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Journal of Medical Internet Research}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{25}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{e45897}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.jmir.org/2023/1/e45897/}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.2196/45897}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{JMIR Publications Toronto, Canada}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100">Conference Proc</abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/AD-480.webp 480w,/assets/img/publication_preview/AD-800.webp 800w,/assets/img/publication_preview/AD-1400.webp 1400w," type="image/webp" sizes="200px"></source> <img src="/assets/img/publication_preview/AD.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="AD.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="ku2023high" class="col-sm-8"> <div class="title">A high-dimensional incomplete-modality transfer learning method for early prediction of Alzheimer’s disease</div> <div class="author"> Dohyun Ku, Zhiyang Zheng, <em>Lingchao Mao</em>, Rui Qi Chen, and <span class="more-authors" title="click to view 7 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '7 more authors' ? 'Yi Su, Kewei Chen, David Weidman, Teresa Wu, Fleming Lure, ShihChung Lo, others' : '7 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">7 more authors</span> </div> <div class="periodical"> <em>Alzheimers and Dementia</em>, 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Prediction of Alzheimer’s disease (AD) risk for individuals with mild cognitive impairment (MCI) provides an opportunity for early intervention. Neuroimaging of different types/modalities has shown promise, but not every patient has all the modalities due to the cost and accessibility constraints. To integrate incomplete multi-modality datasets, we previously developed a machine learning (ML) model called incomplete-modality transfer learning (IMTL). We extended the capacity of IMTL to handle high-dimensional feature sets, namely, HD-IMTL, to further improve accuracy and robustness. Our dataset included 1319 T1-MRI scans from MCI patients in ADNI; among them, 1002 had FDG-PET and 612 had amyloid-PET. 156 regional volumetric and thickness features were computed from MRI and 83 and 83 regional SUVR features from FDG-PET and amyloid-PET, respectively. The dataset is randomly split into training and test sets. The goal of HD-IMTL was to jointly train 4 ML models to predict MCI conversion to AD in 36 months, with each model based on a certain combination of available modalities, namely, MRI, MRI+FDG, MRI+amyloid, and MRI+FDG+amyloid. These correspond to patient sub-cohorts that differ in their access to imaging modalities. To handle high-dimensional features, we employed feature screening to remove uninformative features, performed modality-wise partial least squares (PLS) to condense remaining features into PLS components, and used correlation tests to select components. To jointly train the 4 ML prediction models, IMTL was used, which is a generative model that uses expectation-maximization (EM) in joint parameter estimation to facilitate transfer learning. To account for sample imbalance in training, the Synthetic Minority Over-sampling Technique (SMOTE) was used. The trained models were applied to the test set. 20 training/test splits were repeated and AUCs on the test set were averaged. For comparison, three existing ML models for incomplete-modality fusion were applied to the same dataset. The AUCs by HD-IMTL were 0.802, 0.840, 0.868, and 0.880 for sub-cohorts with MRI, MRI+FDG, MRI+amyloid, and MRI+FDG+amyloid, respectively. The AUCs by existing methods were lower, with ranges of 0.749-0.793, 0.769-0.826, 0.816-0.863, and 0.832-0.868 </p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2022</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100">Conference Proc</abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/uti-480.webp 480w,/assets/img/publication_preview/uti-800.webp 800w,/assets/img/publication_preview/uti-1400.webp 1400w," type="image/webp" sizes="200px"></source> <img src="/assets/img/publication_preview/uti.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="uti.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="mao2022personalized" class="col-sm-8"> <div class="title">Personalized Predictions for Unplanned Urinary Tract Infection Hospitalizations with Hierarchical Clustering</div> <div class="author"> <em>Lingchao Mao</em>, Kimia Vahdat, Sara Shashaani, and Julie L Swann </div> <div class="periodical"> <em></em> 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="award btn btn-sm z-depth-0" role="button">Award</a> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="award hidden d-print-inline"> <p></p> <p>INFORMS ICSS Best Student Paper Finalist, 2020</p> </div> <div class="abstract hidden"> <p>Urinary Tract Infection (UTI) is one of the most frequent and preventable healthcare-associated infections in the US and an important cause of morbidity and excess healthcare costs. This study aims to predict the 30-day risk of a beneficiary for unplanned hospitalization for UTI. Using 2008–12 Medicare fee-for-service claims and several public sources, we extracted 784 features, including patient demographics, clinical conditions, healthcare utilization, provider quality metrics, and community safety indicators. To address the challenge of high heterogeneity and imbalance in data, we propose a hierarchical clustering approach that leverages existing knowledge and data-driven algorithms to partition the population into groups of similar risk, followed by building a LASSO-Logistic Regression (LLR) model for each group. Our prediction models are trained on 237,675 2011 Medicare beneficiaries and tested on 230,042 2012 Medicare beneficiaries. We compare the clustering-based approach to a baseline LLR model using five performance metrics, including the area under the curve (AUC), the True Positive Rate (TPR), and the False Positive Rate (FPR). Results show that the hierarchical clustering approach achieves more accurate and precise predictions (AUC 0.72) than the benchmark model and offers more granular feature importance insights for each patient group.</p> </div> </div> </div> </li></ol> </div> </article> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> © Copyright 2024 Lingchao Mao. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. Last updated: December 16, 2024. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?2781658a0a2b13ed609542042a859126"></script> <script defer src="/assets/js/common.js?e0514a05c5c95ac1a93a8dfd5249b92e"></script> <script defer src="/assets/js/copy_code.js?12775fdf7f95e901d7119054556e495f" type="text/javascript"></script> <script defer src="/assets/js/jupyter_new_tab.js?d9f17b6adc2311cbabd747f4538bb15f"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.2/es5/tex-mml-chtml.js" integrity="sha256-MASABpB4tYktI2Oitl4t+78w/lyA+D7b/s9GEP0JOGI=" crossorigin="anonymous"></script> <script defer src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6" crossorigin="anonymous"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> <script src="/assets/js/vanilla-back-to-top.min.js?f40d453793ff4f64e238e420181a1d17"></script> <script>addBackToTop();</script> </body> </html>